bots/image-refresh

```diff
@@ -135,15 +135,15 @@ def run(image, verbose=False, **kwargs):
     if branch:
         pull = task.pull(branch, **kwargs)
 
+        # we want to validate all image refreshes with external projects
+        task.label(pull, ["test-external"])
+
         # Trigger this pull request
         head = pull["head"]["sha"]
         api = github.GitHub()
         for trigger in triggers:
             api.post("statuses/{0}".format(head), { "state": "pending", "context": trigger,
                 "description": github.NOT_TESTED_DIRECT })
-        # pre-creating statuses circumvents tests-scan's detection of whether bots/ changed
-        # we want to validate all image refreshes with external projects
-        task.label(pull, ["test-external"])
```

Comment:
Please add the "why" to the commit message, not just the "what".

Reply:
ack